### PR TITLE
The first Grid template added.

### PR DESCRIPTION
**Pull request by Sergiy-Dev**
The first grid template added.
3 Columns: Name - Created On - Comments
"Created On" - is filling by random dates for now.
Table is 100% width, just component container max-width = 960px.

### CODE REVIEW

**Review by Eshmann — APPROVED** [review]

File: APIHealthMonitorWeb/src/app/home/home.component.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<mat-table [dataSource]="dataSource" class="mat-elevation-z8">
+
+    <ng-container [matColumnDef]="column" *ngFor="let column of tableColumns; index as i">
+        <mat-header-cell *matHeaderCellDef>{{headers[i]}}</mat-header-cell>
+        <mat-cell *matCellDef="let element">
+            <span *ngIf="column !=='createdOn'">{{element[column]}}</span>
```
Comment (YuraHrysyuk) [inline APIHealthMonitorWeb/src/app/home/home.component.html]: Please remove this and hardcode data as a string with data something like: '07/23/2020'

File: APIHealthMonitorWeb/src/app/home/home.component.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        </mat-cell>
+    </ng-container>
+
+    <mat-header-row *matHeaderRowDef="tableColumns"></mat-header-row>
+    <mat-row *matRowDef="let row; columns: tableColumns;"></mat-row>
+
+</mat-table>
+
+  
+
```
Comment (YuraHrysyuk) [inline APIHealthMonitorWeb/src/app/home/home.component.html]: Please do not add empty lines to the code

File: APIHealthMonitorWeb/src/app/home/home.component.ts [hunk 3]
```diff
@@ -1,19 +1,44 @@
...
+  comments: string;
+}
+
+const DATA_ARRAY: DataElement[] = [
+  {name: 'Jhon', comments: 'Some comment 1'},
+  {name: 'Bob', comments: 'Some comment 2'},
+  {name: 'Kate', comments: 'Some comment 3'},
+  {name: 'Alice', comments: 'Some comment 4'},
+  {name: 'Piter', comments: 'Some comment 5'}
+];
```
Comment (YuraHrysyuk) [inline APIHealthMonitorWeb/src/app/home/home.component.ts]: Please add already formated data to the objects with attr createdOn, like I showed above

File: APIHealthMonitorWeb/src/app/home/home.component.ts [hunk 4]
```diff
@@ -1,19 +1,44 @@
...
     this.titleService.setTitle(this.componentTitle);
   }
 
+  getDate() {
+    const options = {hour: '2-digit', minute: '2-digit', year: 'numeric', month: 'short', day: 'numeric'};
+    const s = new Date(2020, 1, 1).getTime();
+    const f = new Date().getTime();
+    const date = new Date(s + Math.random() * (f - s));
+    return `${date.toLocaleDateString('ru-RU', options)}`;
+  }
```
Comment (YuraHrysyuk) [inline APIHealthMonitorWeb/src/app/home/home.component.ts]: Please do not do such manipulations

**Review by YuraHrysyuk — APPROVED** [review]
Good job man! This one is much better!

**Pull request merged**